packages/rainbowkit/src/wallets/walletConnectors/iopayWallet/iopayWallet.ts

```diff
@@ -27,6 +27,10 @@ export const iopayWallet = ({
       'https://play.google.com/store/apps/details?id=io.iotex.iopay.gp&pli=1',
     ios: 'https://apps.apple.com/us/app/iopay-multichain-crypto-wallet/id1478086371',
     qrCode: 'https://iopay.me/',
+    chrome: 'https://iopay.me/',
+    edge: 'https://iopay.me/',
+    firefox: 'https://iopay.me/',
+    opera: 'https://iopay.me/'
```

Comment:
You'll actually just want to use `browserExtension` here for a fallback URL, but it is recommended that you link to the exact store URLs for each browser